### DISCUSSION TITLE
Change handling of backslash/slash in configs

### DISCUSSION THREAD

**Discussion by patrickpichler** [Dynatrace/dynatrace-configuration-as-code#203] [Archive]
As of right now, all `/` in the config will be replaced with `\` on windows. This
could lead to some strange behaviors, such as replacing `/` in names.

This is done solely to handle paths on windows, but if you think about it, there
is no need to replace every `/` in the config, as we read files from disk only once
at the beginning. Afterwards `/` could always be used to reference other configs.

**Comment by tatomir146** [comment]
Will be solved with monaco config 2.0
